src/map/clif.cpp

```diff
@@ -9225,26 +9225,14 @@ void clif_wedding_effect(struct block_list *bl)
 
 /// Notifies the client of the name of the partner character (ZC_COUPLENAME).
 /// 01e6 <partner name>.24B
-void clif_callpartner(struct map_session_data *sd)
+void clif_callpartner(struct map_session_data& sd, struct map_session_data& p_sd)
 {
-	unsigned char buf[26];
-
-	nullpo_retv(sd);
-
+	unsigned char buf[26]={0};
```

Comment:
Might be best to utilize the packet structure since we are modifying this function.